docs/src/components/PlaygroundRenderSettings.tsx

```diff
@@ -0,0 +1,37 @@
+import clsx from 'clsx';
+import styles from '../pages/playground.module.css';
+
+function PlaygroundRenderSettings({ onSubmit }) {
+  return (
+    <div className={clsx(styles.settings)}>
```

Comment:
```suggestion
    <div className={styles.settings}>
```
clsx is usefull if you merge stuff from styles and classNames